lesson19.md

```diff
@@ -0,0 +1,114 @@
+<!-- .slide: id="lesson19" -->
+
+# Basic Frontend - Spring 2021
+
+Lesson 19, Tuesday, 2021-05-11
+
+---
+
+### Recap: Arrays
+
+An Array is a collection of elements.
+
+Multiple values are stored in a single variable.
+
+Access is index-based.
+
+---
+
+### Quiz
+
+What's printed to console?
+
+```js
+let arr = [ 1, 2, 3 ];
+console.log(arr[1]);
+```
+
+`2`<!-- .element: class="fragment" -->
+
+---
+
+### Array vs. Object
+
+Array access is index based (first index is `0`)
+
+Object access is name based (key-value pairs)
+
+```js
+let arr = [ 'Pizza', 'Muffin' ];
+let obj = {
+    favoriteFood = 'Pizza',
+    favoriteSweet = 'Muffin'
+}
+```
+
+---
+
+### Recap: DOM
+
+What do you remember about DOM?
+
+---
+
+### DOM and Arrays
+
+Let's say we have the following HTML:
+
+```html
+<ul id="todoList">
+  <li>Eat
+  <li>Sleep
+  <li>Work
+</ul>
```

Comment:
isn't it better to properly close them? I know that it's valid html, but I'm sure a few people would ask about it and kind of diverge